src/migrations/M00008.js

```diff
@@ -10,45 +10,6 @@ export async function up(knex) {
 		table.string('appUuid');
 		table.string('commit');
 	});
-
-	const legacyAppsPresent = await knex('config')
-		.where({ key: 'legacyAppsPresent' })
-		.select('value');
-
-	// If there are legacy apps we let the database normalization function
-	// populate the correct values
-	if (legacyAppsPresent && legacyAppsPresent.length > 0) {
```

Comment:
Here is the bug, if the legacy migration ran, then`legacyAppsPresent` would be set to `'false'`